src/app/components/cds/inputs/TextField.module.css

```diff
@@ -70,6 +70,9 @@
     textarea {
       max-width: 100%;
       min-height: 3.5em;
+    }
+
+    .textarea-autoGrow {
```

Comment:
this css definition needs to be `&.textarea-autoGrow` inside of the `textarea {` portion above...so inside of lines 71-72, so it gets applied to the right spot in the dom